docs/supported-languages.md

```diff
@@ -62,8 +124,7 @@ The following thresholds define each maturity level:
     * Parse rate: 99.9%+
     * Rules: 10+
     * Features:
-        * All in experimental
-        * All in beta
+        * All items in Beta
         * `deep_expr_operator`
         * `dots_method_chaining`
         * `equivalence_constant_propagation`
```

Comment:
@s-santillan to unify style with your Supply chain support level tables below, should I convert these three bullet lists with nested lists in lists to three tables?

Reply:
You can try it out and see if you like it!